### PR TITLE
2559 new dataset placeholder text

### LINKED ISSUE #2559: Change placeholder text for new datasets
## Description
Currently if you create a new dataset without drag/dropping a file, the dataset title will show up as "Untitled" by default.

image

It'd be better if the textbox was blank by default, and had a helpful placeholder like "Enter dataset title".

It'd also be nice if there was a bit of explanation around what a good title looks like - e.g. "We recommend ensuring dataset file names are descriptive so users can easily understand the contents". This could fit into the same lightbulb styling as usual.

## Technical Notes
- There are plenty of examples of lightbulb tips - this should be reused, and refactored into a component if we haven't done it already.

## Acceptance Criteria
- The dataset title box isn't prefilled with "Untitled" for new datasets - if a title can't be pre-populated from a file, it should simply be blank
- The dataset title box has a helpful placeholder like "Enter dataset title"
- There's a tip (with a lightbulb icon) with the text "We recommend ensuring dataset file names are descriptive so users can easily understand the contents" above the title textbox.

### PR DESCRIPTION
**Pull request by sajidanower23**
### What this PR does

Fixes #2559 

- Makes the placeholder title for a new dataset blank
- Adds a tooltip for the title

### Checklist

-   [x] There are unit tests to verify my changes are correct or unit tests aren't applicable
-   [x] I've updated CHANGES.md with what I changed.
-   [x] I've linked this PR to an issue in ZenHub (core dev team only)

maga_untitled_tooltip

### CODE REVIEW

**Comment by t83714** [comment]
Read the issue description and it sounds like the `tip` should be the `lightbulb` style:
image
Rather than a mouse-over tooltip: 
image
@alyce Could you please help to confirm this?
if you want to have a play, the test site is here:
https://2559-new-dataset-placeholder-text.dev.magda.io

**Comment by alyce** [comment]
Yes @t83714 I think you're right, this is the type of content that is better suited for a lightbulb. Anything that says "we recommend..." is one of our general data literacy and capability points so they're better in the lightbulb style.

**Comment by sajidanower23** [comment]
@alyce @t83714  used a lightbulb icon. This is what it looks like now. Thoughts?
lightbulb_icon

**Comment by sajidanower23** [comment]
@t83714  Here is the latest tooltip
lightbulb_tooltip

File: CHANGES.md [hunk 1]
```diff
@@ -119,6 +119,9 @@ UI:
 -   Fixed CSV loader didn't retry the different line ending correctly
 -   Saved publisher id in the database
 -   Added hover text for the tooltip beside the date-picker in the Add Dataset page
+-   Added a distribution hyperlink to the title of resource links
```
Comment (t83714) [inline CHANGES.md]: When you merge the master, this line has been deplicated

Comment (sajidanower23) [reply CHANGES.md]: Oh good catch!

**Comment by sajidanower23** [comment]
When there is a title
title

When there is no title
no_title

**Comment by t83714** [comment]
@alyce 
I think it's ready for a final check (in terms of looking and feel) ---- no rush~ 
Here is how it looks like now:
image
If you want to see it in action, here is the link:
https://2559-new-dataset-placeholder-text.dev.magda.io/dataset/add

**Comment by alyce** [comment]
Looks good to me!

**Pull request merged**